lib/model/content.dart

```diff
@@ -450,6 +450,10 @@ class StrongNode extends InlineContainerNode {
   const StrongNode({super.debugHtmlNode, required super.nodes});
 }
 
+class DeletedNode extends InlineContainerNode {
```

Comment:
I am not sure about the name here it seems ambiguous. Any Suggestions will be very welcomed!

Reply:
I think I see what you mean — it sounds like it might mean a node that's been deleted somehow, like it's no longer part of the tree.

Alternative names could be `DelNode`, or `StrikethroughNode`.

I don't like the name `DelNode`, though — it feels pretty opaque. We don't name other node types based on abbreviations like that; e.g. we say `EmphasisNode`, not `EmNode`.

And although `StrikethroughNode` could be reasonable if we made a pattern of naming these based on how we're going to render them, we don't — we instead name them based on their HTML meaning, so e.g. `StrongNode` rather than `BoldNode`. It'd be awkward to have a mixture of those patterns.

So let's go with `DeletedNode`, as it seems the most canonical name here to parallel `EmphasisNode` and others. I think the confusion shouldn't be too much of a problem, because we don't actually have a concept like "deleting" a node to confuse it with. If we did, we might have to rename to avoid that confusion.

Reply:
Thanks for you thoughts, I totally agree.